lib/ai_bot/entry_point.rb

```diff
@@ -102,6 +105,14 @@ def inject_into(plugin)
           Rails.root.join("plugins", "discourse-ai", "db", "fixtures", "ai_bot"),
         )
 
+        plugin.add_to_serializer(
+          :topic_view,
+          :is_bot_pm,
+          include_condition: -> do
+            object.personal_message && object.topic.custom_fields[TOPIC_AI_BOT_PM_FIELD]
```

Comment:
What about existing PMs?

Reply:
@xfalcox Doesn't mess with them.. they have an existing bot PM. This would leave room in the future for us to somehow backfill.. but this is really to address the case of 
* PM is sent
* Navigate to PM
* There is no response yet.. we don't have any clue it's a bot PM (and the mechanisms to determine can have false-positive like the case with auto-responder)

I didn't remove the conditionals for checking if there is a bot PM, so old PMs still function.

Reply:
So we are adding a permanent custom field just to address the temporary case of detection during the first transition? I don't really agree littering the database permanently is a good strategy in this case.

Can't this be handled by a JS state, URL param, or anything else that has a more appropriate lifetime ?